app.js

```diff
@@ -0,0 +1,14 @@
+const express = require("express");
+const { getTopics } = require("./controllers/topics.controller");
+const app = express();
+
+//GET
+app.get("/api/topics", getTopics);
+
+//Error Handling
```

Comment:
You may look to add handling for a 404 here, should you add this (rather than rely on default express 404 error) You will need to add testing for it